devbuild.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -eu
+
+#cd to location of script
+MYDIR=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+
+usage () {
+  echo "Usage: "
+  echo "  ./build.sh PLATFORM COMPILER"
```

Comment:
Should this message say "./devbuild.sh ..." instead of "./build.sh ..."?

Reply:
Thanks for pointing that out. I have updated the script to use the builtin $0 variable instead (script name)